ocs_sample_library_preview/DataView/Field.py

```diff
@@ -1,26 +1,33 @@
 import json
 
 from .FieldSource import FieldSource
+from ..SDS.SdsSummaryType import SdsSummaryType
+from .SummaryDirection import SummaryDirection as SummaryDirectionType
 
 
 class Field(object):
     """OCS Field definition"""
 
     def __init__(self, source: FieldSource = None, keys: list[str] = None,
                  stream_reference_names: list[str] = None, label: str = None,
-                 include_uom: bool = None):
+                 include_uom: bool = None, summary_type: SdsSummaryType = SdsSummaryType.none,
```

Comment:
It looks like we generally set ```summary_type``` to ```SdsSummaryType = None``` (See DataMapping.py and FieldMapping.py)

Reply:
so that was kind of my question though in the chat the other day. I don't know what these mapping objects are or where they're used, but I don't believe this code works. I started with this code exactly, and it fails when trying to deserialize in the fromJson() method.

Reply:
actually this is a better response for your next comment than this one. and i agree on both. i changed this back to None for consistency's sake